db/01_create_chitter_table.sql

```diff
@@ -0,0 +1 @@
+REATE TABLE chitter (id SERIAL PRIMARY KEY, peep VARCHAR(60)); 
```

Comment:
Need the C on CREATE
And might want to adjust the VARCHAR to 240 instead of 60 so the peep can be longer?